### ISSUE TITLE
Segmentation results

### ISSUE THREAD

**Issue by sheshap** [j1a0m0e4sNTU/3dgcn#6]
I tried to recreate segmentation results but could achieve only
Best mIoU(c): 0.811, Best mIoU (i): 0.846

Please let me know if any changes need to be done to achieve the results in the paper
i.e. mIoU(c): 0.821, mIoU (i): 0.851

Thanks in Advance.

**Comment by chih-hao-lin** [comment]
I think you can try to run the experiments with more epochs (about 150), or load the trained model weights and train again with a smaller learning rate (= 5e-5). 
Hope this helps.
